src/extension.ts

```diff
@@ -13,6 +13,14 @@ export function activate(context: vscode.ExtensionContext) {
 	// Trigger the Manim shell to start listening to the terminal
 	ManimShell.instance;
 
+	const showVersionCommand = vscode.commands.registerCommand(
+		'manim-notebook.showVersion', () => {
+			Logger.info(`💠 Command requested: Show Version`);
+			let version = context.extension.packageJSON.version;
+			Window.showInformationMessage(`Manim Notebook version: ${version}`);
+		});
+
```

Comment:
Thanks for this command. I just realized that I already print the current Manim notebook version via the `logSystemInformation()` method in the `logger.ts`. And there I read the JSON file manually, your method is way more elegant! I will adopt that for the `logSystemInformation` as well.

Do we still want to keep your `showVersion` command? The only use case I see is when people don't attach a log file in the issues template. But even then, they should! What about just always requiring people to record a small log file (even if they don't want to reproduce anything), just because it also records some other system information that could be useful.

Reply:
And on top, your code was not for vain since I learnt about `context.extension.packageJSON.version;` and could use that in `logSystemInformation` (instead of having to read the `package.json` file via a file system process 🙈

Reply:
from my side, feel free to resolve this thread